pkgs/by-name/kc/kcl-language-server/package.nix

```diff
@@ -9,18 +9,18 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "kcl-language-server";
-  version = "0.11.2";
+  version = "0.11.2.68b4062";
```

Comment:
Sorry, only noticed this now: This is not how we do unstable versioning in Nixpkgs.

Please make it follow the contribution guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning